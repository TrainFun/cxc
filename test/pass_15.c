int main() {
  write cast<double>(123);
  write cast<double>(true);
  write cast<int>(.);
  write cast<int>(true) + 1145;
  write cast<bool>(123);
  write cast<bool>(124);
  write cast<bool>(.);
}
