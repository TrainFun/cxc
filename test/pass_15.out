123.000000
1.000000
0
1146
1
1
0
